lightningd/options.c

```diff
@@ -1285,7 +1285,7 @@ static char *opt_set_codex32(const char *arg, struct lightningd *ld)
 	if (fd < 0) {
 		/* Don't do anything if the file already exists. */
 		if (errno == EEXIST)
-			return tal_fmt(tmpctx, "HSM_Secret already exists!");
+			return tal_fmt(tmpctx, "hsm_secret already exists!");
 
 		return tal_fmt(tmpctx, "Creating hsm_secret: %s",
```

Comment:
Belongs in previous commit?

Reply:
Yes...

Reply:
Yes, Thanks for pointing out. I'll fix it now.